insights/client/auto_config.py

```diff
@@ -66,6 +66,11 @@ def set_auto_configuration(config, hostname, ca_cert, proxy):
         saved_proxy = config.proxy
         config.proxy = proxy
     config.base_url = hostname + '/r/insights'
+    if ca_cert is None:
```

Comment:
I think it would have been clearer to pass in an explicit 'is_satellite_managed' flag  since strictly speaking  direct connected systems also have a CA cert, its just that we handle it differently.  I am ok with shipping this as is though.

Reply:
agreed